### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by jamestucker1988**

### CODE REVIEW

**Comment by jamestucker1988** [comment]
the project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

**Comment by jamestucker1988** [comment]
I could not figure out my linter/stylelint error. In my local machine there is no stylelint errors. please help me.

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
-# tnw
\ No newline at end of file
+# Project 5 - smashing_magazine_clone
+​
+This is an assigned project where we were supposed to replicate/clone smashingmagazine page as seen in the link:
+<a href="https://www.smashingmagazine.com/">click here</a>
+​
+​
```
Comment (elmejdki) [inline README.md:6]: - please your readme file professional, this is not a smashing magazine clone. fix the links also

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
...
+
+## Preview of our webpage
```
Comment (elmejdki) [inline README.md:8]: remove the unused titles.

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    name: Stylelint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x
```
Comment (bolah2009) [inline .github/workflows/linters.yml:45]: Can you please update your config to the recent configuration? 🙂
This should solve the issue you are having. Sorry for the inconvenience. 🙏🏾

Please check here https://github.com/microverseinc/linters-config/blob/07433021cd211c0a22c51a2cb797e86e25d2dde2/html-css/.github/workflows/linters.yml#L45


## 💡 THIS IS NOT A CRR

Comment (bolah2009) [reply .github/workflows/linters.yml:45]: @jamestucker1988 the config file, you should change is here 👇 (**.github/workflows/linters.yml**)

https://github.com/jamestucker1988/tnw/blob/7838b7da0dc0a1fed4ba4bbd6155717068216249/.github/workflows/linters.yml#L45

```diff
-          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
```

Comment (jamestucker1988) [reply .github/workflows/linters.yml:45]: thank u so much bolah . I greatly appreciate ur help.  otherwise, I would have been stuck there.

**Comment by jamestucker1988** [comment]
the project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

File: README.md [hunk 4]
```diff
@@ -1 +1,63 @@
-# tnw
\ No newline at end of file
+# Project 5 - smashing_magazine_clone
```
Comment (expjazz) [inline README.md:1]: The title is wrong.

**Comment by jamestucker1988** [comment]
the project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

**Comment by jamestucker1988** [comment]
The project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

**Here is screen shot of webpage (4 row of images on my desktop)**<a href="images/tnwscreenshot.png">click here</a>

**Comment by krys2fa** [comment]
## Great job implementing the changes requested, @jamestucker1988 👍  
Your project displays good use of semantic HTML as well as good CSS best practices. 💪 

Just a few more adjustments to complete the project.
### Status: Changes Required ♻️ 

Please note that these changes are requested at a breakpoint of 768px and 1024px
- [ ] The icons in the footer have rounded borders and the links below them are all capitalized.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
Screen Shot 2020-07-19 at 7 37 58 AM |  Screen Shot 2020-07-19 at 7 38 15 AM


- [] Your page looks different on both breakpoints. Below are the differences at breakpoint 768px.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
Screen Shot 2020-07-19 at 7 43 49 AM |  Screen Shot 2020-07-19 at 7 44 03 AM


#### HINT: You can use your browser's developer tools to test the appearance of your clone across multiple screen resolutions.
You can then compare your changes with the original site.

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 
Chris Amanor (Slack)

**Comment by jamestucker1988** [comment]
fixed the changes mentioned after review.
1)footer icons have rounded borders.
2)the breakpoint issue has been adjusted.

**Comment by jamestucker1988** [comment]
### THE NEXT WEB CLONE
- Introduced the breakpoint 1024 px in the media query.
-styled background of footer icons.
- adjusted the latest-deal section.
-adjusted the latest-news section.
-adjusted nav-bar.

**Comment by jamestucker1988** [comment]
### THE NEXT WEB CLONE.
-In this pull request I have fixed the following enumerated in bullets.
- On the top 3 news, the TWN logo disappears on medium size:
image

-On the Latest Funding Section, the image should be inside the boxes for all sizes:
image
-The text from Latest Founding Section overlaps with the categories section when close to medium size.
-Text is not properly aligned inside the images on the Sections links.

**Comment by jamestucker1988** [comment]
### THE NEXT WEB CLONE.
-In this pull request, I have fixed the following enumerated in bullets
-At 756 px, the menu bar and the logo have been fixed.
-improved the header section at 756px width.
-Improved font-size of the funding section.

File: README.md [hunk 5]
```diff
@@ -1 +1,56 @@
...
+​
+Give a ⭐️ if you like this project!
+​
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+  ​
```
Comment (Spayco) [inline README.md:51]: - [ ] either fill these or just delete the section

File: index.html [hunk 6]
```diff
@@ -0,0 +1,926 @@
...
+              ></span></h3
+          ></a>
+          <div class="high-five">
+            <h5>Add info</h5>
+            <h5>|</h5>
+            <h5>Get updates</h5>
+            <h5>|</h5>
+            <h5>see More</h5>
+          </div>
+        </div>
```
Comment (Spayco) [inline index.html:234]: - [ ] on the medium and small screens, the texts are flipped 
website:
image
image

yours:

image
image

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello There! 👋 

Awesome job so far!! 👍 you're few steps closer to approval

## Required Changes ♻️ 

* [ ] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)), instead of commenting before every review, simply put the description on your first post in the PR

- [ ] your page has some horizontal overflow, please fix it 
image

- ** I included more requested changes below**

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by jamestucker1988** [comment]
This project is a curriculum-based work where we have to clone  thenextweb.com. 
- I have used modern css techniques like grid and flex to style the page.
-appropriate use of semantics is considered.
-the project is divided into nav, main-section, footer with sub-section.

**Review by Spayco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!

Tip: From now on please include your description in the top post, not in a separate comment


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**